declarative-examples/buildPluginParentPOM.groovy

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    tools {
+        maven "apache-maven-3.1.0"
+        jdk "default"
+    }
+
+    agent {
+        label "master"
+    }
+
+    stages {
+        stage("build") {
```

Comment:
I prefer simple lowercase names, but that may be a personal preference. Lowercased texts are proved to be easier to read than capitalized ones.